body-pix/demos/index.js

```diff
@@ -570,6 +579,15 @@ async function estimatePartSegmentation() {
   return allPersonPartSegmentation;
 }
 
+async function getAndSetBackend() {
+  const backend = getBackendFromQueryString();
+
+  if (backend === 'cpu') {
```

Comment:
see comment above. Define a constant CPU_STRING

Reply:
Better to not have the data type in the variable name...I'll just do CPU